gammapy/makers/background/fov.py

```diff
@@ -31,20 +36,30 @@ class FoVBackgroundMaker(Maker):
     spectral_model : SpectralModel or str
         Reference norm spectral model to use for the `FoVBackgroundModel`, if none is defined
         on the dataset. By default, use pl-norm.
+    min_counts : float
```

Comment:
Strictly speaking this would be an integer number...

Reply:
Ok changed. I had kept it float because we do not enforce the requirement of `int` anywhere. This `min_counts` requirement can come as a result of some calculation, which can turn out to be float...

Reply:
In practice it doesn't matter anyway. However it might be good to mention whether it's greater or greater equal...